packages/client/hmi-client/src/components/widgets/tera-asset-nav.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+	<nav>
+		<a v-for="content in assetContentIdsInView" :key="content" @click="scrollTo(content)">
+			{{ content.replace('-', ' ') }}
+		</a>
+	</nav>
+</template>
+
+<script setup lang="ts">
+import { ref, onMounted, nextTick } from 'vue';
+
+const props = defineProps<{
+	assetContentIds: string[];
+}>();
+
+const assetContentIdsInView = ref<string[]>([]);
+
+async function scrollTo(elementId: string) {
+	const element = document.getElementById(elementId);
+	if (!element) return;
+	element.scrollIntoView({ behavior: 'smooth' });
+}
```

Comment:
`document.getElementById` is slightly dangerous because you don't know how other folks will name things....otherwise we may want some prefix or suffix to decrease the chances of id/name collisions.

Would it be possible to pass in the section-container to tera-asset-nav, and do getElementId on that instead of the entire document?

```
const props = defineProps<{
	assetContentIds: string[];
  containerElement?: HTMLElement
}>();

... blahblahblah ...

const element = props.containerElement ? containerElement.getElementById(elemetnId) : document.getElementById(elementId);
```